### PR TITLE
tests, logs, h2 db, postman screen, readme screen

### PR DESCRIPTION
**Pull request by u2107680007911543774e7r**

### CODE REVIEW

File: office-management-app/src/main/java/saadow/office_management_app/controller/OfficeController.java [hunk 1]
```diff
@@ -30,19 +31,12 @@
 
 
 	@Autowired
-	private OfficeService officeService;
+	private OfficeService officeService = new OfficeServiceImpl();
```
Comment (Yanetta) [inline office-management-app/src/main/java/saadow/office_management_app/controller/OfficeController.java]: What is the reason to use @Autowired and new at the same time?

Comment (SerhiiPasko) [reply office-management-app/src/main/java/saadow/office_management_app/controller/OfficeController.java]: Yes, it critical

File: office-management-app/src/main/resources/log4j2.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+			fileName="logs${sys:file.separator}example-logger.log"
+			filePattern="info-%d{yyyy-MM-dd}-%i.log">
+			<PatternLayout pattern="${LOG_PATTERN}" />
+			<Policies>
+				<SizeBasedTriggeringPolicy size="30 MB" />
+			</Policies>
+			<DefaultRolloverStrategy max="2" />
+		</RollingFile>
+    
+		<SMTP name="FEMAIL" SMTPProtocol="smtps"
```
Comment (SerhiiPasko) [inline office-management-app/src/main/resources/log4j2.xml]: Where in code you use this logger appender?

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read comments

**Review by Yanetta — APPROVED** [review]
Good!

**Pull request merged**